easybuild/easyblocks/x/xalt.py

```diff
@@ -0,0 +1,92 @@
+##
+# Copyright 2020 NVIDIA
+#
+# This file is triple-licensed under GPLv2 (see below), MIT, and
+# BSD three-clause licenses.
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for XALT, implemented as an easyblock
+@author: Scott McMillan (NVIDIA)
+"""
+import os
+import re
+
+from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from easybuild.framework.easyconfig import CUSTOM, MANDATORY
+from easybuild.tools.build_log import EasyBuildError
+
+
+class EB_XALT(ConfigureMake):
+    """Support for building and installing XALT."""
+
+    @staticmethod
+    def extra_options():
+        extra_vars = {
+            'config_py': [None, "XALT site filter file", MANDATORY],
+            'syshost': ['hardcode:cluster', "System name", CUSTOM],
```

Comment:
We should leave `syshost` unspecified, and also require that a value is specified (similar to `config_py`), since there's no way we can guess a good default value (and hardcoding `cluster` is a bad idea imho).

The error message produced should point to https://xalt.readthedocs.io/en/latest/020_site_configuration.html ?